lib/util.ts

```diff
@@ -27,23 +27,14 @@ type ThriftObject = NewFileMetaData | parquet_thrift.PageHeader | parquet_thrift
   * names for every PageLocation
   */
 
-// Issue at https://github.com/LibertyDSNP/parquetjs/issues/42
-const previousPageLocation = new parquet_thrift.PageLocation();
-//@ts-ignore
-const PageLocation = parquet_thrift.PageLocation.prototype = [];
-//@ts-ignore
-PageLocation.write = previousPageLocation.write;
```

Comment:
Can we safely get rid of the read/writes?  I assume you checked.

Reply:
The current test suite is passing, but I can write more tests for the Reader class to be sure.

Reply:
Yes I tested against the ts conversion for reader and writer and there were no complaints